python/ray/rllib/evaluation/sampler.py

```diff
@@ -34,6 +34,11 @@
     "prev_reward"
 ])
 
+# Bonsai Constants
```

Comment:
Let's removed the Bonsai wording.

Reply:
Why? This is a bonsai-only change that should not be committed back to Ray. It seems to make sense to me to note that in the comments.

Reply:
If it's a bonsai only change then why does this change need to exist here? Why not keep it in our machine Teaching repo instead?

Reply:
Uhh, what? Did you not participate in PDP2 development? Do you not remember the pain of moving entirely new files into our repo?

Reply:
This 6 line change would be an absolute pain to make in the MT repo. We've never touched the sampler before.

Reply:
Thanks Ross for the work! As we chatted on Teams, how about doing a custom call-backs here so we can merge it? I can also help.